src/MongoDB.Driver.Core/Core/Servers/ServerMonitor.cs

```diff
@@ -128,7 +128,7 @@ public void Initialize()
         {
             if (_state.TryChange(State.Initial, State.Open))
             {
-                _ = MonitorServerAsync().ConfigureAwait(false);
+                _ = Task.Factory.StartNew(() => MonitorServerAsync().ConfigureAwait(false)).ConfigureAwait(false);
```

Comment:
the only new change

Reply:
For consistency should this be?

    _ = Task.Factory.StartNew(() => _ = MonitorServerAsync().ConfigureAwait(false)).ConfigureAwait(false);

Reply:
For that matter I don't think the calls to ConfigureAwait are needed, so this could just be:

    _ = Task.Factory.StartNew(() => _ = MonitorServerAsync());
    _ = _roundTripTimeMonitor.RunAsync();

But maybe we don't want to risk making this change.

Reply:
>_ = Task.Factory.StartNew(() => _ = MonitorServerAsync().ConfigureAwait(false)).ConfigureAwait(false);

done

>But maybe we don't want to risk making this change.

this was my idea about this change. I would leave it as is